lib/sem/credentials.rb

```diff
@@ -0,0 +1,13 @@
+module Sem
+  class Credentials
+    PATH = File.expand_path("~/.sem/credentials").freeze
+
+    def self.write(auth_token)
+      File.write(PATH, auth_token)
```

Comment:
We need to make sure that the file has the correct permissions. `0600` seems correct or `06400`.